firebase.json

```diff
@@ -24,1610 +24,648 @@
-      },
-        {
-        "source": "/:lang/docs/guides/pwa-amp/amp-to-pwa",
-        "destination": "/:lang/docs/integration/pwa-amp/amp-to-pwa.html",
+        "source": "/:lang/docs/getting_started/visual_story/setting_up",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/start/visual_story/setting_up?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/guides/responsive_amp",
-        "destination": "/:lang/docs/design/responsive_amp.html",
+        "source": "/:lang/docs/getting_started/visual_story/start_story",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/start/visual_story/start_story?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/guides/responsive/art_direction",
-        "destination": "/:lang/docs/design/responsive/art_direction.html",
+        "source": "/:lang/docs/getting_started/visual_story/validation",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/start/visual_story/validation?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/guides/responsive/control_layout",
-        "destination": "/:lang/docs/design/responsive/control_layout.html",
+        "source": "/:lang/docs/integration/",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/guides/responsive/custom_fonts",
-        "destination": "/:lang/docs/design/responsive/custom_fonts.html",
+        "source": "/:lang/docs/integration/integrate-amphtml-email",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/contribute/integrate-amphtml-email?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/guides/responsive/placeholders",
-        "destination": "/:lang/docs/design/responsive/placeholders.html",
+        "source": "/:lang/docs/integration/integrate-with-apps",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/integrate/integrate-with-apps?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/guides/responsive/responsive_design",
-        "destination": "/:lang/docs/design/responsive/responsive_design.html",
+        "source": "/:lang/docs/integration/integrate-your-tech",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/contribute/integrate-your-tech?referrer=ampproject.org",
         "type": 301
       },
       {
-        "source": "/:lang/docs/guides/responsive/style_pages",
-        "destination": "/:lang/docs/design/responsive/style_pages.html",
+        "source": "/:lang/docs/integration/pwa-amp",
+        "destination": "https://amp.dev/:lang/documentation/guides-and-tutorials/integrate/amp-in-pw
```

Comment:
this should go to https://amp.dev/documentation/guides-and-tutorials/develop/media_iframes_3p/amp_replacements.html?format=websites

Reply:
Did you probably slip lines during review? This pair currently is a 1:1 match.

Reply:
Really? The articles in https://www.ampproject.org/docs/media/  are the same as in  https://amp.dev/documentation/guides-and-tutorials/develop/media_iframes_3p